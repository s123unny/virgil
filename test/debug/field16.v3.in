b ../core/field16.v3 34
b ../core/field16.v3 35
b ../core/field16.v3 36
b ../core/field16.v3 37
b ../core/field16.v3 38
b ../core/field16.v3 39
b ../core/field16.v3 40
b ../core/field16.v3 41
b ../core/field16.v3 42
b ../core/field16.v3 46
run
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
bt
info l
c
q
