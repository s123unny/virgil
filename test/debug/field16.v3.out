breakpoint|../core/field16.v3 34|0
breakpoint|../core/field16.v3 35|1
breakpoint|../core/field16.v3 36|2
breakpoint|../core/field16.v3 37|3
breakpoint|../core/field16.v3 38|4
breakpoint|../core/field16.v3 39|5
breakpoint|../core/field16.v3 40|6
breakpoint|../core/field16.v3 41|7
breakpoint|../core/field16.v3 42|8
breakpoint|../core/field16.v3 46|9
stop|stopOnBreakpoint
bt|main|../core/field16.v3|34
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|35
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|36
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|37
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|38
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|39
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|40
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|41
variable|0|arg|0|int|false
stop|stopOnBreakpoint
bt|main|../core/field16.v3|42
variable|0|arg|0|int|false
stop|end
result|77
